frontend/src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { UserContext } from '../../UserContext.js';
+import './LoginForm.css'
+
+const LoginForm = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const { updateUser } = useContext(UserContext);
+
+  const navigate = useNavigate();
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+
+    try {
+      // Make the login API request
+      const response = await fetch(`http://localhost:3000/users/login`, {
```

Comment:
Probably want to make sure this will support production use case too instead of just `localhost`.

Reply:
To do

Reply:
Done